doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -891,3 +893,4 @@ Documentation
 * Added the :ref:`log_rpc` library documentation page.
 * Removed the Getting started with nRF7002 DK and Getting started with other DKs pages from the :ref:`gsg_guides` section.
   These pages were no longer relevant as the `Quick Start app`_ now also supports the nRF7002 DK.
+* The Wi-Fi credentials library documentation has been moved to the upstream zephyr.
```

Comment:
Can we add a link to the doc here? Also, move it before the previous bullet "Removed the Getting started.....".
```suggestion
* Moved the Wi-Fi credentials library page to the upstream Zephyr repository.
```